src/Components/Board/Board.js

```diff
@@ -0,0 +1,19 @@
+import React, { useContext } from 'react';
+import { UserContext } from '../../Context/GameContext.js';
+import Box from '../Box/box.js';
+import './Board.css';
+
+export default function Board() {
+  const { boxes, turn, winnerMessage } = useContext(UserContext);
+  return (
+    <>
+      <h1>{winnerMessage}</h1>
+      <h2>{turn}s Turn</h2>
```

Comment:
perhaps you could use the ```active``` state variables and do something like ```
active && <h2>{turn}s Turn</h2>``` that way the h2 would only render if the game is active so it doesn't say "Xs turn" even if the game is over